release.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+function help() {
+    echo "Usage: sh release.sh <RELEASE_VERSION>"
+    echo ""
+    echo "Parameters:"
+    echo "  RELEASE_VERSION: SemVer Version number for the release"
+}
+
+semver_pattern="^(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)$"
+
+RELEASE_VERSION=${1}
+
+[[ -z "${RELEASE_VERSION}" ]] && echo "ERROR: RELEASE_VERSION is empty" && help  && exit 1
+[[ ! "${RELEASE_VERSION}" =~ $semver_pattern ]] && echo "ERROR: RELEASE_VERSION does not match the SemVer specification" && help && exit 1
+
```

Comment:
To ensure to be in sync with origin, I'd suggest to fetch first.

```suggestion

git fetch
```